prototype_source/pt2e_quant_ptq_static.rst

```diff
@@ -0,0 +1,444 @@
+(prototype) PyTorch 2.0 Export Post Training Static Quantization
+================================================================
+**Author**: `Jerry Zhang <https://github.com/jerryzh168>`_
+
+This tutorial introduces the steps to do post training static quantization in graph mode based on
+`torch._export.export <https://pytorch.org/docs/main/export.html>`_. Compared to `FX Graph Mode Quantization <https://pytorch.org/tutorials/prototype/fx_graph_mode_ptq_static.html>`, this flow is expected to have significantly higher model coverage (`88% on 14K models <https://github.com/pytorch/pytorch/issues/93667#issuecomment-1601171596>`), better programmability, and a simplified UX.
```

Comment:
@svekars looks like you are making some final updates, could you add `_` to "`FX Graph Mode Quantization <https://pytorch.org/tutorials/prototype/fx_graph_mode_ptq_static.html>`" as well?